package.json

```diff
@@ -4,8 +4,13 @@
   "description": "Some sample web app.",
   "main": "index.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "live-server"
+    "build-task:scss-compile": "node-sass-chokidar --source-map true _scss/style.scss -o _temp",
+    "build-task:autoprefixer": "postcss _temp/style.css --use autoprefixer -d _css",
```

Comment:
`build-task:autoprefixer` Runs `postcss` on _temp/style.css, feeding it through `autoprefixer` and spitting it out into `_css/style.css`.